src/_pytest/compat.py

```diff
@@ -342,6 +342,12 @@ def final(f):
         return f
 
 
+# TODO: sealed classes are not supported yet
+# https://github.com/mypyc/mypyc/issues/846
+def sealed(f: _T) -> _T:
```

Comment:
This is no longer used and can be removed?

Reply:
Yep